### PR TITLE
Compass to inquiry pr

### PR DESCRIPTION
**Pull request by pavankumar0408**
Changes have been added for Examination feature. In add library, filters list of questions based on evaluation mode 'server' or 'client'

### CODE REVIEW

**Comment by rajnishdargan** [comment]
@pavankumar0408 Please change the package version to `"version": "7.0.0-beta.0"` here - https://github.com/Sunbird-inQuiry/sunbird-resource-library/blob/release-7.0.0/projects/resource-library/package.json#L3

Note - Player web component version should also be updated in the package.json file. For reference: https://github.com/Sunbird-inQuiry/sunbird-resource-library/commit/0330a01ba0fed4bdbe198e9057e4b68200de4880

**Comment by pavankumar0408** [comment]
updated

**Comment by pavankumar0408** [comment]
@rajnishdargan , package has been updated

File: projects/resource-library/package-lock.json
Comment (rajnishdargan) [inline projects/resource-library/package-lock.json:1]: @pavankumar0408 This file is not needed, please delete this file.

File: package-lock.json
Comment (rajnishdargan) [inline package-lock.json:1]: @pavankumar0408 
`lockfileVersion` is getting change from `"lockfileVersion":1` to `"lockfileVersion":2`
Please use node version 14 to update the package-lock.json file.

**Review by rajnishdargan — APPROVED** [review]

**Review by sajeshkayyath — APPROVED** [review]

**Pull request merged**